ocp_resources/datavolume.py

```diff
@@ -252,8 +252,16 @@ def pvc(self):
 
     @property
     def scratch_pvc(self):
+        scratch_pvc_prefix = (
+            f"prime-{self.pvc.instance.metadata.uid}"
+            if self.pvc.instance.metadata.annotations.get(
```

Comment:
`self.pvc.instance.metadata` - save and re-use

Reply:
ack